llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1333,6 +1333,15 @@ def Experimental
    : SubtargetFeature<"experimental", "HasExperimental",
                       "true", "Experimental intrinsics">;
 
+def FeatureABIVLen32B
```

Comment:
Can we put this in the frontend's CodeGenOpts instead of using feature flags? I don't really want to burn 12 bits in SubtargetFeatures on something the backend doesn't need.

I think you can use a CC1Option with MarshallingInfo in Options.td to define a command line flag for the cc1 process to get it into CodeGenOpts.